server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1634,9 +1633,9 @@ void trimOperations(long belowTerm, long aboveSeqNo) {
             final Set<Long> seenSeqNo = new HashSet<>();
 
             int size = operationsList.size();
-            return IntStream.range(0, size + 1)
-                // current + reverse traverse of operations
-                .mapToObj(i -> i == 0 ? currentOperations : operationsList.get(size - i))
+            return IntStream.range(0, size)
```

Comment:
why just use operationList.stream() ?

Reply:
I'm sorry. got confused. nevermind

Reply:
That said, instead of being fancy here, we can do this on add and existing ops if they have the same seq# but a lower term. That will improve the test.